CHANGELOG.md

```diff
@@ -2,62 +2,64 @@
 
 ## master / unreleased
 
+
+## 0.6.0 / 2020-01-22
+
 Note that the ruler flags need to be changed in this upgrade. You're moving from a single node ruler to something that might need to be sharded.
 If you are running with a high `-ruler.num-workers` and if you're not able to execute all your rules in `-ruler.evaluation-interval`, then you'll need to shard.
```

Comment:
This text can go, I think.  It was based on an incorrect diagnosis.

Reply:
Done.